includes/definitions.inc.php

```diff
@@ -1435,6 +1435,19 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+// Lenovo EMC (NAS)
+$os = 'lenovoemc';
```

Comment:
BTW, FYI - Lenovo EMC seems to be a branch of Lenovo (e.g. http://setup.lenovoemc.com/setup.php)